sdk/storage/azure-storage-file-datalake/out/azure/storage/filedatalake/_upload_helper.pyi

```diff
@@ -0,0 +1,11 @@
+# Stubs for azure.storage.filedatalake._upload_helper (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._deserialize import process_storage_error
+from ._generated.models import StorageErrorException
+from ._shared.response_handlers import return_response_headers
+from ._shared.uploads import DataLakeFileChunkUploader, upload_data_chunks
+from typing import Any, Optional
+
+def upload_datalake_file(client: Optional[Any] = ..., stream: Optional[Any] = ..., length: Optional[Any] = ..., overwrite: Optional[Any] = ..., validate_content: Optional[Any] = ..., max_concurrency: Optional[Any] = ..., **kwargs: Any): ...
```

Comment:
There are too many positional parameters to this function.

Reply:
I believe this is an internal function